Apps/Examples/Examples/All Examples/ShowHideLayerExample.swift

```diff
@@ -0,0 +1,164 @@
+        // can be found at https://docs.mapbox.com/vector-tiles/reference/mapbox-terrain-v2/
+        contourSource.url = "mapbox://mapbox.mapbox-terrain-v2"
+
+        var contourLayer = LineLayer(id: contourLayerId)
+
+        // Assign this layer's source and source layer ID.
+        contourLayer.source = contourSourceId
+        contourLayer.sourceLayer = "contour"
+
+        // Style the contents of the source's contour layer.
+        contourLayer.lineCap = .constant(.round)
+        contourLayer.lineJoin = .constant(.round)
+
+        // `visibility` is `nil` by default. Set to `visible`.
+        contourLayer.visibility = .constant(.visible)
+        let contourLineColor = UIColor(red: 0.53, green: 0.48, blue: 0.35, alpha: 1.00)
+        contourLayer.lineColor = .constant(ColorRepresentable(color: contourLineColor))
+
+        let style = mapView.mapboxMap.style
+
+        // Add the sources and layers to the map's style.
+        do {
+            try style.addSource(museumsSource, id: museumSourceId)
+            try style.addSource(contourSource, id: contourSourceId)
+            try style.addLayer(museumLayer)
+            try style.addLayer(contourLayer)
+        } catch {
+            print("Error when adding sources and layers: \(error.localizedDescription)")
+        }
+    }
+
+    @objc func toggleMuseumLayerVisibility(sender: UISwitch) {
+        let style = mapView.mapboxMap.style
+        // Update the museum layer's visibility based on whether the switch
+        // is on. `visibility` is `nil` by default.
+        do {
+            try style.updateLayer(withId: museumLayerId) { (layer: inout CircleLayer) in
+                layer.visibility = .constant(sender.isOn ? .visible : .none)
+            }
+        } catch {
+            print("Failed to update the visibility for layer with id \(museumLayerId). Error: \(error.localizedDescription)")
+        }
+    }
+
+    @objc func toggleContourLayerVisibility(sender: UISwitch) {
+        let style = mapView.mapboxMap.style
+        // Update the contour layer's visibility based on whether the switch
+        // is on.
+        do {
+            try style.updateLayer(withId: contourLayerId) { (layer: inout CircleLayer) in
+                layer.visibility = .constant(sender.isOn ? .visible : .none)
+            }
+        } catch {
+            print("Failed to update the visibility for layer with id \(contourLayerId). Error: \(error.localizedDescription)")
+        }
+    }
+
+    func addVisibilitySwitches() {
+        // Create switches to toggle the layers' visibility.
+        let museumSwitch = UISwitch()
+        museumSwitch.addTarget(self, action: #selector(toggleMuseumLayerVisibility(sender:)), for: .valueChanged)
+        museumSwitch.isOn = true
+        museumSwitch.translatesAutoresizingMaskIntoConstraints = false
+        view.insertSubview(museumSwitch, aboveSubview: mapView)
```

Comment:
All of these `insertSubview` calls can just be `addSubview` which adds the new view on top by default